modules/Bio/EnsEMBL/IO/Format.pm

```diff
@@ -289,14 +333,12 @@ sub validate_as_boolean {
 
 sub validate_as_string {
   my ($self, $value, $match) = @_;
-  my $error;
   if ($match) {
-    $error = "String $value does not match $match" unless $value eq $match;
+    return $value eq $match ? 1 : 0;
   }
   else {
-    $error = "Value $value is not a string" unless $value =~ /[[:print:]]+/;
+    return $value =~ /[[:print:]]+/ ? 1 : 0;
```

Comment:
I would really prefer using Regexp::Common here instead of reinventing the wheel, especially considering this is may or may not be Unicode-safe (IIRC it depends on the Perl version used).